.github/workflows/tests.yml

```diff
@@ -0,0 +1,51 @@
+name: Tests
+
+on:
+  push:
+  pull_request:
+  schedule:
+    - cron: '0 0 * * *'
+
+jobs:
+  tests:
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: true
+      matrix:
+        php: [7.2, 7.3, 7.4, 8.0]
+        stability: [prefer-lowest, prefer-stable]
+
+    name: PHP ${{ matrix.php }} - ${{ matrix.stability }}
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Setup PHP
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: ${{ matrix.php }}
+          extensions: dom, curl, libxml, mbstring, zip
+          coverage: none
+
+      - name: Install dependencies
+        run: composer update --${{ matrix.stability }} --prefer-dist --no-interaction --no-progress
+
+#      - name: Run PHP CS Fixer
+#        if: ${{ matrix.php != 8.0 }}
+#        run: |
+#          composer require friendsofphp/php-cs-fixer:^2.15 --dev --prefer-dist --no-interaction --no-progress --update-with-all-dependencies
+#          vendor/bin/php-cs-fixer fix -v --dry-run --using-cache=no
```

Comment:
I've removed PHP CS Fixer from `composer.json` because it's not yet patched for PHP 8. I'm now installing it when that step is run.

However, I've disabled it for now as it was failing. When I tried to patch the changes by running it without `--dry-run` it started adding type-hints which seemed like breaking changes to me. So this might be something you want to look into yourself soon.